processor/src/main/java/org/mapstruct/ap/services/DefaultAccessorNamingStrategy.java

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright 2012-2015 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.services;
+
+import java.beans.Introspector;
+
+import org.mapstruct.spi.AccessorNamingStrategy;
+
+/**
+ * The default JavaBeans-compliant implementation of the {@link AccessorNamingStrategy} service provider interface.
+ *
+ * @author Christian Schuster
+ */
+class DefaultAccessorNamingStrategy implements AccessorNamingStrategy {
```

Comment:
I also thought about that. Future default implementations might need classes from `mapstruct-processor`, which would then cause a circular maven dependency to/from `mapstruct-spi`. So I'd rather keep the implementation secret and document the default behavior in the interface's Javadoc.

I thought about some kind of `setDefaultAccessorNamingStrategy()` that is invoked by the `Services` class before returning the custom implementation.